cerializer/tests/benchmark.py

```diff
@@ -0,0 +1,222 @@
+import timeit
+from typing import List, Union, Dict, Hashable, Any, Tuple
+
+import texttable
+import yaml
+import cerializer.cerializer_handler
+import cerializer.tests.test_cerializer_compatibility
+import cerializer.utils
+import logwood
+
+
+SCHEMA_ROOT = '/Users/matejmicek/PycharmProjects/Cerializer/cerializer/tests/schemata'
+SCHEMA_ROOT = '/Users/matejmicek/PycharmProjects/schema_root'
+
+SCHEMA_ROOTS = [SCHEMA_ROOT]
+
+
+def benchmark_schema_deserialize(
+	schema_roots: List[str],
+	schema_favro: Union[Dict[Hashable, Any], list, None],
+	path: str,
+	count: int,
+	schema_name: str,
+	schema_version: int,
+	namespace: str,
+	schema_identifier: str,
+) -> Tuple:
+	'''
+	Helper function. This should not be used on its own. Use benchmark() instead.
```

Comment:
We usually prefix the function with `_` if it should not be used. Then it is a "private" function